src/Controls/src/Core/Platform/Windows/CollectionView/GroupedItemTemplateCollection.cs

```diff
@@ -15,6 +15,11 @@ internal class GroupedItemTemplateCollection : ObservableCollection<GroupTemplat
 		readonly BindableObject _container;
 		readonly IMauiContext _mauiContext;
 		readonly IList _groupList;
+		readonly NotifyCollectionChangedEventHandler _collectionChanged;
+		readonly WeakNotifyCollectionChangedProxy _proxy = new();
+		bool _disposedValue;
+
+		~GroupedItemTemplateCollection() => _proxy.Unsubscribe();
```

Comment:
On Dispose you do a null check with `?` operator... Wouldn't make sense to do it here as well?